src/test/java/org/influxdb/InfluxDBProxyTest.java

```diff
@@ -0,0 +1,77 @@
+package org.influxdb;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.influxdb.dto.Point;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.QueryResult;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+/**
+ * Test the InfluxDB API.
+ *
+ * @author hoan.le [at] bonitoo.io
+ *
+ */
+@RunWith(JUnitPlatform.class)
+public class InfluxDBProxyTest {
+  private InfluxDB influxDB;
+  private String db = "udp";
```

Comment:
The database should be named via-proxy or so, but not udp, this is misleading

Reply:
change naming
BTW, name of the database for UDP is configured at server side